src/main/java/org/spongepowered/api/world/extent/BiomeArea.java

```diff
@@ -30,10 +30,31 @@
 
 /**
  * An area containing biomes.
+ *
+ * <p>Some methods accept a pair of two ints, representing the x and z location
+ * of a biome. Some other methods accept or return a {@link Vector2i}. The
+ * y position of this vector is equal to the z position of a biome in the
+ * world.</p>
  */
```

Comment:
That's a concern outside the scope of this PR, see the resolution of #70 (decided to use a vector2 in the end).